lib/src/render.dart

```diff
@@ -17,7 +16,8 @@ String render(String path,
     {bool color: false,
     SyncPackageResolver packageResolver,
     bool useSpaces: true,
-    int indentWidth: 2}) {
+    int indentWidth: 2,
+    LineFeed linefeed: LineFeed.LF}) {
```

Comment:
Generally we avoid providing default argument values for non-boolean arguments, since it makes it impossible to explicitly say "use the default value". You can just use `??` in the method body instead.

Same goes for `indentWidth`, now that I think of it.